src/styles/index.scss

```diff
@@ -1,3 +1,117 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  width: 198px;
+  height: 406px;
```

Comment:
You should not hardcode height value. Elements inside of the card should manipulate the height

Reply:
link to new pull requests: https://github.com/mate-academy/layout_product-cards/pull/4959

Reply:
<img width="635" alt="image" src="https://github.com/user-attachments/assets/c39c5bb1-f0bb-4a51-8051-5e0b7d09618e">